lib/runner/cli/cli.js

```diff
@@ -318,6 +339,18 @@ class CliRunner {
     return this;
   }
 
+  get isMobile() {
+    for (const env of this.testEnvArray) {
+      const {platformName} = this.testEnvSettings[env].desiredCapabilities || {};
+
+      if ((platformName && (platformName.toLowerCase() === 'ios' || platformName.toLowerCase() === 'android'))) {
+        return true;
+      }
```

Comment:
@gravityvi Because we are not passing `platformName` in android's config so this change will force us to pass `platformName` capability from config and due to this `create-nightwatch `and `mobile-helper` will require to update. Instead, there is already a capability available to identify the android which is `androidPackage`.

```suggestion
      const {desiredCapabilities} = this.testEnvSettings[env];
      const options = desiredCapabilities['goog:chromeOptions'] || desiredCapabilities['moz:firefoxOptions'];
      
      if ((options && options.androidPackage)) {
        this.testEnvSettings[env].desiredCapabilities.platformName = 'android';
        return true;
      }
      
      const {browserName = '', platformName = ''} = desiredCapabilities;
      
      if (browserName.toLowerCase() === 'safari' && platformName.toLowerCase() === 'ios'){
        return true;
      }
```